syntax = "proto3";

package matcher_proto;

message Book {
  optional int64 id = 1;
}
